client/app/app.js

```diff
@@ -0,0 +1,22 @@
+
+var util = require('../../server/utilities.js');
+
+angular.module('GitSecureResults', [])
+
+.factory('results', function() {
+  var instance = {};
+  var counts;
+
+  instance.getResults = function() {
+  
+  counts = util.getCounts();
+  
+  return instance;
+})
+
+.controller('showResults', ['$scope', 'results', function($scope, results) {
+
+  $scope.results = function() {
+    $scope.display = results.getResults();
```

Comment:
My initial thought is that we should just do

`$scope.display = util.getCounts()` 

and drop the factory function.

Looking at util.getCounts() though this module is broken as it uses to async calls and should not be returning an immediate variable.